specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01/afdx.json

```diff
@@ -4931,12 +5005,17 @@
           }
         },
         "queryStringCachingBehavior": {
-          "description": "Defines how AzureFrontDoor caches requests that include query strings. You can ignore any query strings when caching, bypass caching to prevent requests that contain query strings from being cached, or cache every request with a unique URL.",
-          "$ref": "./cdn.json#/definitions/QueryStringCachingBehavior"
-        },
-        "optimizationType": {
```

Comment:
We are removing this property, and causing a breaking change alert on this removal. 
Could we explain why?

Reply:
This was because this field is not used, and since this is a new service, we want to remove it before it gets out.